content-src/components/TopSites/TopSites.jsx

```diff
@@ -139,7 +139,7 @@ export class _TopSites extends React.PureComponent {
         <div className="edit-topsites-wrapper">
           {editForm &&
             <div className="edit-topsites">
-              <div className="modal-overlay" onClick={this.onEditFormClose} />
+              <div className="modal-overlay" onClick={this.onEditFormClose} role="presentation" />
```

Comment:
This is correct, although in addition to this I think it might be worth filing a bug (not under this metabug, just under http://as-bugzy.herokuapp.com/feature/1550082 with the `access` keyword) to support the escape button for closing all modals